### DISCUSSION TITLE
Is there a way to track unused fields

### DISCUSSION THREAD

**Discussion by prakashkvau** [webonyx/graphql-php#1384] [Q&A]
Hey, is it possible to either log/track all the unused fields on the graph easily?

we have a large graph and want to track which fields are not used by the clients.

Thanks!

**Comment by spawnia** [comment]
We offer a way to track usage of deprecated fields with Lighthouse, see https://github.com/nuwave/lighthouse/blob/499f3c0f8686195a15dde92fb8ef4cf387e701ad/docs/master/digging-deeper/deprecation.md. This class could serve as a template for you to implement your own tracking for every field: https://github.com/nuwave/lighthouse/blob/499f3c0f8686195a15dde92fb8ef4cf387e701ad/src/Deprecation/DetectDeprecatedUsage.php.
